Tests/CommandsTests/PackageToolTests.swift

```diff
@@ -2554,4 +2554,72 @@ final class PackageToolTests: CommandsTestCase {
             }
         }
     }
+
+    func testSinglePluginTarget() throws {
```

Comment:
Moved to a different file (same content except the test name) and added XCTSkipIf line